src/Sylius/Bundle/MailerBundle/Renderer/Adapter/TwigAdapter.php

```diff
@@ -62,7 +64,7 @@ public function render(EmailInterface $email, array $data = array())
         }
 
         /** @var EmailEvent $event */
-        $event = $this->dispatcher->dispatch(self::EVENT_EMAIL_RENDERED, new EmailEvent(new RenderedEmail($subject, $body)));
+        $event = $this->dispatcher->dispatch(SyliusMailerEvents::EMAIL_RENDERED, new EmailRenderEvent(new RenderedEmail($subject, $body)));
```

Comment:
We have pre/post UPDATE/CREATE/DELETE so I think this should be RENDER. What do you think?